### PR TITLE
Add google ads audience APIs

### PR DESCRIPTION
**Pull request by aashikam**
## Purpose
$title

### CODE REVIEW

File: en/docs/reference/connectors/google-ads-connector/google-ads-configuration.md [hunk 1]
```diff
@@ -714,6 +714,210 @@ To use the Google Ads connector, first create the connection with your configura
     }
     ```
 
+??? note "campaignCriteriaMutate"
+    Creates, updates, or removes criteria. Operation statuses are returned.
```
Comment (SanojPunchihewa) [inline en/docs/reference/connectors/google-ads-connector/google-ads-configuration.md:718]: Shall we follow the same format as https://github.com/wso2/docs-mi/pull/1003/files#diff-c21ad6e833416d1a7b70bc9961744e37e6da384d18f2016df0a0d9bf43ac0859L655

File: en/docs/reference/connectors/google-ads-connector/google-ads-configuration.md [hunk 2]
```diff
@@ -714,6 +714,210 @@ To use the Google Ads connector, first create the connection with your configura
...
+    <table>
+        <tr>
+            <th>Parameter Name</th>
+            <th>Description</th>
+            <th>Required</th>
+        </tr>
+        <tr>
+            <td>customerId</td>
```
Comment (DinithiDiaz) [inline en/docs/reference/connectors/google-ads-connector/google-ads-configuration.md:726]:
```suggestion
            <td><code>customerId</code></td>
```

File: en/docs/reference/connectors/google-ads-connector/google-ads-configuration.md [hunk 3]
```diff
@@ -714,6 +714,210 @@ To use the Google Ads connector, first create the connection with your configura
...
+            <td>The ID of the customer whose criteria are being modified.</td>
+            <td>Yes</td>
+        </tr>
+        <tr>
+            <td>operations</td>
```
Comment (DinithiDiaz) [inline en/docs/reference/connectors/google-ads-connector/google-ads-configuration.md:731]:
```suggestion
            <td><code>operations</code></td>
```

File: en/docs/reference/connectors/google-ads-connector/google-ads-configuration.md [hunk 4]
```diff
@@ -714,6 +714,210 @@ To use the Google Ads connector, first create the connection with your configura
...
+            <td>The list of operations to perform on individual criteria.</td>
+            <td>Yes</td>
+        </tr>
+        <tr>
+            <td>partialFailure</td>
```
Comment (DinithiDiaz) [inline en/docs/reference/connectors/google-ads-connector/google-ads-configuration.md:736]:
```suggestion
            <td><code>partialFailure</code></td>
```

File: en/docs/reference/connectors/google-ads-connector/google-ads-configuration.md [hunk 5]
```diff
@@ -714,6 +714,210 @@ To use the Google Ads connector, first create the connection with your configura
...
+            <td>If true, successful operations will be carried out and invalid operations will return errors. If false, all operations will be carried out in one transaction if and only if they are all valid. Default is false.</td>
+            <td>No</td>
+        </tr>
+        <tr>
+            <td>validateOnly</td>
```
Comment (DinithiDiaz) [inline en/docs/reference/connectors/google-ads-connector/google-ads-configuration.md:741]:
```suggestion
            <td><code>validateOnly</code></td>
```

File: en/docs/reference/connectors/google-ads-connector/google-ads-configuration.md [hunk 6]
```diff
@@ -714,6 +714,210 @@ To use the Google Ads connector, first create the connection with your configura
...
+            <td>If true, the request is validated but not executed. Only errors are returned, not results.</td>
+            <td>No</td>
+        </tr>
+        <tr>
+            <td>responseContentType</td>
```
Comment (DinithiDiaz) [inline en/docs/reference/connectors/google-ads-connector/google-ads-configuration.md:746]:
```suggestion
            <td><code>responseContentType</code></td>
```

File: en/docs/reference/connectors/google-ads-connector/google-ads-configuration.md [hunk 7]
```diff
@@ -714,6 +714,210 @@ To use the Google Ads connector, first create the connection with your configura
...
+??? note "customAudiencesMutate"
+    Creates or updates custom audiences. Operation statuses are returned.
+    <table>
+        <tr>
+            <th>Parameter Name</th>
+            <th>Description</th>
+            <th>Required</th>
+        </tr>
+        <tr>
+            <td>customerId</td>
```
Comment (DinithiDiaz) [inline en/docs/reference/connectors/google-ads-connector/google-ads-configuration.md:788]:
```suggestion
            <td><code>customerId</code></td>
```

File: en/docs/reference/connectors/google-ads-connector/google-ads-configuration.md [hunk 8]
```diff
@@ -714,6 +714,210 @@ To use the Google Ads connector, first create the connection with your configura
...
+            <td>The ID of the customer whose custom audiences are being modified.</td>
+            <td>Yes</td>
+        </tr>
+        <tr>
+            <td>operations</td>
```
Comment (DinithiDiaz) [inline en/docs/reference/connectors/google-ads-connector/google-ads-configuration.md:793]:
```suggestion
            <td><code>operations</code></td>
```

File: en/docs/reference/connectors/google-ads-connector/google-ads-configuration.md [hunk 9]
```diff
@@ -714,6 +714,210 @@ To use the Google Ads connector, first create the connection with your configura
...
+            <td>The list of operations to perform on individual custom audiences.</td>
+            <td>Yes</td>
+        </tr>
+        <tr>
+            <td>validateOnly</td>
```
Comment (DinithiDiaz) [inline en/docs/reference/connectors/google-ads-connector/google-ads-configuration.md:798]:
```suggestion
            <td><code>validateOnly</code></td>
```

File: en/docs/reference/connectors/google-ads-connector/google-ads-configuration.md [hunk 10]
```diff
@@ -714,6 +714,210 @@ To use the Google Ads connector, first create the connection with your configura
...
+??? note "audiencesMutate"
+    Creates audiences. Operation statuses are returned.
+    <table>
+        <tr>
+            <th>Parameter Name</th>
+            <th>Description</th>
+            <th>Required</th>
+        </tr>
+        <tr>
+            <td>customerId</td>
```
Comment (DinithiDiaz) [inline en/docs/reference/connectors/google-ads-connector/google-ads-configuration.md:854]:
```suggestion
            <td><code>customerId</code></td>
```

File: en/docs/reference/connectors/google-ads-connector/google-ads-configuration.md [hunk 11]
```diff
@@ -714,6 +714,210 @@ To use the Google Ads connector, first create the connection with your configura
...
+            <td>The ID of the customer whose audiences are being modified.</td>
+            <td>Yes</td>
+        </tr>
+        <tr>
+            <td>operations</td>
```
Comment (DinithiDiaz) [inline en/docs/reference/connectors/google-ads-connector/google-ads-configuration.md:859]:
```suggestion
            <td><code>operations</code></td>
```

File: en/docs/reference/connectors/google-ads-connector/google-ads-configuration.md [hunk 12]
```diff
@@ -714,6 +714,210 @@ To use the Google Ads connector, first create the connection with your configura
...
+            <td>The list of operations to perform on individual audiences.</td>
+            <td>Yes</td>
+        </tr>
+        <tr>
+            <td>partialFailure</td>
```
Comment (DinithiDiaz) [inline en/docs/reference/connectors/google-ads-connector/google-ads-configuration.md:864]:
```suggestion
            <td><code>partialFailure</code></td>
```

File: en/docs/reference/connectors/google-ads-connector/google-ads-configuration.md [hunk 13]
```diff
@@ -714,6 +714,210 @@ To use the Google Ads connector, first create the connection with your configura
...
+            <td>If true, successful operations will be carried out and invalid operations will return errors. If false, all operations will be carried out in one transaction if and only if they are all valid. Default is false.</td>
+            <td>No</td>
+        </tr>
+        <tr>
+            <td>validateOnly</td>
```
Comment (DinithiDiaz) [inline en/docs/reference/connectors/google-ads-connector/google-ads-configuration.md:869]:
```suggestion
            <td><code>validateOnly</code></td>
```

File: en/docs/reference/connectors/google-ads-connector/google-ads-configuration.md [hunk 14]
```diff
@@ -714,6 +714,210 @@ To use the Google Ads connector, first create the connection with your configura
...
+            <td>If true, the request is validated but not executed. Only errors are returned, not results.</td>
+            <td>No</td>
+        </tr>
+        <tr>
+            <td>responseContentType</td>
```
Comment (DinithiDiaz) [inline en/docs/reference/connectors/google-ads-connector/google-ads-configuration.md:874]:
```suggestion
            <td><code>responseContentType</code></td>
```

File: en/docs/reference/connectors/google-ads-connector/google-ads-configuration.md [hunk 15]
```diff
@@ -714,6 +714,210 @@ To use the Google Ads connector, first create the connection with your configura
...
+            <td>Yes</td>
+        </tr>
+        <tr>
+            <td>operations</td>
+            <td>The list of operations to perform on individual criteria.</td>
+            <td>Yes</td>
+        </tr>
+        <tr>
+            <td>partialFailure</td>
+            <td>If true, successful operations will be carried out and invalid operations will return errors. If false, all operations will be carried out in one transaction if and only if they are all valid. Default is false.</td>
```
Comment (DinithiDiaz) [inline en/docs/reference/connectors/google-ads-connector/google-ads-configuration.md:737]:
```suggestion
            <td>If true, successful operations will be carried out and invalid operations will return errors. If false, all operations will be carried out in one transaction if and only if they are all valid. The default value is false.</td>
```

File: en/docs/reference/connectors/google-ads-connector/google-ads-configuration.md [hunk 16]
```diff
@@ -714,6 +714,210 @@ To use the Google Ads connector, first create the connection with your configura
...
+            <td>Yes</td>
+        </tr>
+        <tr>
+            <td>operations</td>
+            <td>The list of operations to perform on individual audiences.</td>
+            <td>Yes</td>
+        </tr>
+        <tr>
+            <td>partialFailure</td>
+            <td>If true, successful operations will be carried out and invalid operations will return errors. If false, all operations will be carried out in one transaction if and only if they are all valid. Default is false.</td>
```
Comment (DinithiDiaz) [inline en/docs/reference/connectors/google-ads-connector/google-ads-configuration.md:865]:
```suggestion
            <td>If true, successful operations will be carried out and invalid operations will return errors. If false, all operations will be carried out in one transaction if and only if they are all valid. The default value is false.</td>
```

**Review by arunans23 — APPROVED** [review]

**Pull request merged**